src/Renci.SshNet/SshClient.cs

```
@@ -313,7 +331,7 @@ public Shell CreateShell(Stream input, Stream output, Stream extendedOutput, str
         /// <exception cref="SshConnectionException">Client is not connected.</exception>
         public Shell CreateShell(Stream input, Stream output, Stream extendedOutput)
         {
-            return CreateShell(input, output, extendedOutput, string.Empty, 0, 0, 0, 0, terminalModes: null, 1024);
+            return CreateShell(input, output, extendedOutput, 1024);
```

This looks like a behavioural breaking change. Not sure if we should take it

---

Previously, it will swallow the exception if failed to send pseudo terminal request when start `Shell`. (Note it will throw exception when instantiate `ShellStream`)

---

That's good. What I mean is that this now will not allocate the pseudo terminal where previously it did, and it would not be nice to cause a behaviour change there (if the behaviour is indeed different)